system/Session/Session.php

```diff
@@ -36,59 +38,59 @@ class Session implements SessionInterface
     protected $driver;
 
     /**
-     * The storage driver to use: files, database, redis, memcached
+     * The session storage driver to use.
      *
      * @var string
      */
-    protected $sessionDriverName;
+    protected $handler = FileHandler::class;
 
     /**
-     * The session cookie name, must contain only [0-9a-z_-] characters.
+     * The name of the session which is used as cookie name.
+     * It should only contain alphanumeric characters.
      *
      * @var string
      */
-    protected $sessionCookieName = 'ci_session';
+    protected $name = 'ci_session';
 
     /**
      * The number of SECONDS you want the session to last.
-     * Setting it to 0 (zero) means expire when the browser is closed.
+     * Set to `0` means expire when the browser is closed.
      *
      * @var int
      */
-    protected $sessionExpiration = 7200;
+    protected $lifetime = 7200;
 
     /**
-     * The location to save sessions to, driver dependent..
+     * The location to save sessions to and is driver dependent.
      *
      * For the 'files' driver, it's a path to a writable directory.
      * WARNING: Only absolute paths are supported!
      *
      * For the 'database' driver, it's a table name.
-     *
-     * @todo address memcache & redis needs
+     * Please read up the manual for the format with other session drivers.
      *
      * IMPORTANT: You are REQUIRED to set a valid save path!
      *
      * @var string
      */
-    protected $sessionSavePath;
+    protected $savePath = WRITEPATH . 'session';
 
     /**
      * Whether to match the user's IP address when reading the session data.
      *
-     * WARNING: If you're using the database driver, don't forget to update
-     * your session table's PRIMARY KEY when changing this setting.
+     * NOTE: If you're using the database driver, don't forget to update
+     *       your session table's PRIMARY KEY when changing this setting.
      *
      * @var bool
      */
-    protected $sessionMatchIP = false;
+    protected $matchIP = false;
 
     /**
      * How many seconds between CI regenerating the session ID.
      *
      * @var int
      */
-    protected $sessionTimeToUpdate = 300;
+    protected $cooldown = 300;
```

Comment:
I'm not sure `cooldown` is proper naming. 
At least I don't understand cooldown means session regenerating time.